keras_cv/models/csp_darknet.py

```diff
@@ -40,6 +40,7 @@ def CSPDarkNet(
     num_classes=None,
```

Comment:
lets add a unit test to models_test.py to cover this and make sure we can construct a model (like the backbone) using this API.

Reply:
Done -- I've added a test and verified that without this new parameter, the test fails.